doc/nrf/known_issues.rst

```diff
@@ -687,7 +688,7 @@ The combination of nRF Secure Immutable Bootloader and MCUBoot fails to upgrade
 Build system
 ============
 
-.. rst-class:: v1-5-1 v1-5-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0
```

Comment:
This will never be fixed as for as I can tell, should it be documented somewhere else?

Reply:
I don't think so, this is the place where people will go to see if they hit a known issue and for the workaround.
So even if it means adding this tag for every release, I would still keep it here.